Kickstarter-iOS/ViewModels/UpdatePreviewViewModelTests.swift

```diff
@@ -38,7 +37,7 @@ final class UpdatePreviewViewModelTests: TestCase {
     self.vm.inputs.viewDidLoad()
 
     let previewUrl = "https://\(Secrets.Api.Endpoint.production)/projects/2/updates/1/preview"
-    let query = "client_id=\(self.apiService.serverConfig.apiClientAuth.clientId)"
+    let query = "client_id=\(self.apiService.serverConfig.apiClientAuth.clientId)&currency=USD"
```

Comment:
Right now our way of sending device currency (from the region) is to include it as a param. This means web views should get currency for free. Alternatively we could have synced currency via config, but this could have slowed things during load.